chia/daemon/server.py

```diff
@@ -340,8 +340,7 @@ async def handle_message(
         destination = message["destination"]
         if destination != "daemon":
             if destination in self.connections:
-                sockets = self.connections[destination]
-                return dict_to_json_str(message), sockets
+                return dict_to_json_str(message), self.connections[destination].copy()
```

Comment:
It's not visible in this patch, but your change suggests that there is at least one caller of `handle_message()` that will iterate over the returned set with `async` calls interleaved. Is that right?
I think it would make sense to add a comment here to remind future readers of this code why we need to copy this set. We don't have a general policy to always copy containers or objects we returns (even though it would probably be more robust), so this is a bit special in that way.

Reply:
Yes, see the stack trace in the PR description. There you see `send_all_responses` gets passed in the `connections` received from `handle_message` and in there we iterate over it and then remove the connection if it fails. I changed the place where we copy now with the force push. Instead of copying in `handle_message` it now copies when we actually iterate/remove which means we copy unnecessarily in few cases but only single connection sets, so i think its worth the more obvious way of copying.

Reply:
This appears to be resolved